mkt/developers/tests/test_utils_.py

```diff
@@ -1,3 +1,9 @@
+import os
+
+from django.conf import settings
+
+from nose.tools import ok_
+
 import mkt.site.tests
 from mkt.developers.utils import check_upload
 from mkt.site.utils import LocalFileStorage
```

Comment:
I just submitted a commit that moves this to `mkt.site.storage_utils`. There's also `local_storage` you can use instead of instantiating this below.